packages/web/components/ConnectToProgress.tsx

```diff
@@ -38,130 +38,134 @@ export const ConnectToProgress: React.FC<{
   if (connected && !fetching) {
     if (chainId !== '0x1' && showSwitchButton) {
       return (
-        <Stack color="white" spacing={8} w="100%" maxW="30rem">
-          <MetaGameLogo />
-          {header && <MetaHeading>{header}</MetaHeading>}
-          <Text fontSize="md" w="100%" textAlign="center">
-            Please switch to <SwitchNetworkButton chainId="0x1" /> to progress
-          </Text>
+        <Stack w="100%" align="center">
+          <Stack color="white" spacing={8} w="100%" maxW="30rem">
```

Comment:
Is having two singly nested `Stack`s different from having one with all the attributes?

```suggestion
        <Stack
          w="100%" align="center" color="white"
          spacing={8} maxW="30rem"
        >
```

Reply:
In this situation, yes. I needed the inner `stack` to be centered to the outer `stack` but have a max width of 30rem